UI/html/admin/viewRedFlags.html

```diff
@@ -42,76 +41,76 @@
 
       <div class="right w-100">
         <div class="breadboard">
-          <h2 class="breadboard-header j-c-sb">
-            <span class="head">Red-flags</span>
+          <div class="breadboard-header j-c-sb mg-t-b">
+            <h3 class="w-75 t-c mg-0">All Red-flag records</h3>
             <input type="search" name="searchbar" placeholder="Search here" class="search">
-          </h2>
+          </div>
 
           <div class="dip">
             <ul class="list-item">
               <li class="item list">
                 <div class="date t-c">18<br> Jan</div>
                 <div class="j-c-cl">
-                  <a href="./viewSingleRedFlag.html" class="pd-l"><b>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut,</b></a>
+                  <a href="./changeStatus.html" class="pd-l"><b>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut,</b></a>
                   <div class="pd-l">
                     <small class="pd-r-sm">200 views</small>
-                    <small class="pd-r-sm">status: <i>unresolved</i></small>
```

Comment:
Any reason why you're using all lowercase? I believe this and the other words should be Pascal Case so `Unresolved` instead of `unresolved`